railgun/manager/run.go

```diff
@@ -297,6 +315,9 @@ func Run(ctx context.Context, c *config.Config) error {
 			},
 		}
 		controllers = append(controllers, controller)
+	} else {
+		setupLog.Info(`ingresses.networking.internal.knative.dev v1alpha1 CRD not available on cluster.
+		Disabling Knative controller`)
```

Comment:
configure Knative Controller as enabled
disable it if CRD does not exist
re-enable controller when CRD get deployed. 
I think this is consistent with 1.3.0 behavior.